onnx/__init__.py

```diff
@@ -198,6 +205,16 @@ def save_model(proto: Union[ModelProto, bytes], f: Union[IO[bytes], str], format
     _save_bytes(s, f)
 
 
+def save_lib_proto(proto: Union[LibProto, bytes], f: Union[IO[bytes], str]) -> None:
+    '''
+    '''
+    if isinstance(proto, bytes):
+        proto = _deserialize(proto, LibProto())
```

Comment:
Why deserialize and serialize back?

Reply:
Good point. TODO add so that I can make the codel change when working with LibProto - I need to merge this branch in order to build ort with my optional input branch.